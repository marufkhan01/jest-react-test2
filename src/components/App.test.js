import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../components/App';

it('App component should render as expected',() =>{
    const component = shallow(<App />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});



