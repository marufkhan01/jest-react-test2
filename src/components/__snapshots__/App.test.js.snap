// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render as expected 1`] = `
<div>
  <Login
    loginSuccessful={[Function]}
  />
</div>
`;
